components/productions/index/search.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <pub-input
     class="searchbox"
+    name="productions-overview-search"
```

Comment:
I think @jdrieghe had a comment about `name` being a bad choice of name for this prop because it means something else in HTML. Maybe `id`?

Reply:
See https://github.com/cultuurnet/udb3-frontend/pull/66#discussion_r488866894

Reply:
`id` like `name` or `class` also already has a meaning in html

Reply:
Yes but `id` is would be the same as what it becomes in the HTML. It's more predictable.

Reply:
but maybe `id` matches better with the use of id in plain html, while `name` in html is used for a different reason than identifying solely

Reply:
Exactly 😄

Reply:
done